pkg/util/netlinkops.go

```diff
@@ -0,0 +1,121 @@
+// +build linux
+
+package util
```

Comment:
Can this belong to its own package instead of util?
Something like netlink (and you use the named import where you are consuming this package)

Reply:
Do you mean just importing netlink as another named import?